lib/pr_checker.js

```diff
@@ -249,10 +252,30 @@ class PRChecker {
           cli.warn(infoMsg);
         }
       }
+
+      if (commits.length > 0) {
+        const lastCommit = commits[commits.length - 1].commit;
+        const { status } = lastCommit;
+        switch (status.state) {
+          case EXPECTED:
+          case PENDING:
+            cli.warn('CI test results are still pending');
+            break;
+          case SUCCESS:
+            cli.info('All CI tests are successful');
+            break;
+          case FAILURE:
+            cli.warn('Some or all CI tests seems to be failing or flasky');
```

Comment:
Nit: `s/seems/seem/`, `s/flasky/flaky/`

Reply:
After thinking about it like a brief message for case `FAILURE` because the goal was if the ci is all green they do not have to go and check the CI but when it is `FAILURE` it either because of flasky or unrelated error so i think this should work:
`CI tests have some failures, perhaps maybe unrelated or flasky error.`